locales/cs/experiments.ftl

```diff
@@ -21,7 +23,6 @@ activitystreamContributors11Title = Softwarový inženýr
 activitystreamContributors12Title = Senior UX designér
 cliqzDescription = Rychlejší výsledky vyhledávání. Cliqz zobrazuje návrhy už během psaní přímo pod adresním řádkem. Bez snížení nebo omezení vašeho soukromí.
 cliqzIntroduction = Rychlejší výsledky vyhledávání přímo v prohlížeči. Začněte psát a hned uvidíte návrhy výsledků přímo pod adresním řádkem. A protože Cliqz nesbírá osobní data, která by vás mohla identifikovat, ani nevytváří váš osobní profil, nijak neohrozí vaše soukromí.
-cliqzMeasurements = <p>Experiment Cliqz v programu Test Pilot vznikl ve spolupráci Cliqz GmbH a Mozilly. Cliqz je navržen pro ochranu vašeho soukromí omezením sběru citlivých informací, které mohou být použity k vytvoření vašeho uživatelského profilu a vašeho sledování. Nicméně úplná anonymizace takovýchto dat je velmi složitá a stále může být možná vás částečně identifikovat.</p> <p>Kromě <a href="/privacy">údajů</a> shromažďovaných experimenty programu Test Pilot byste měli o Cliqz také vědět:</p> <p>Cliqz GmbH sbírá data o vašem vyhledávání a prohlížení, včetně:<ul><li>Textu, který zadáte do adresního řádku</li> <li>Dotazů, které pošlete do jednotlivých vyhledávačů</li> <li>Informace o tom, jak používáte webové stránky, pohyby myší, scrollování a strávený čas</li> </ul></p> <p>Cliqz GmbH sbírá telemetrické údaje včetně:<ul><li>Vašich interakcí s konkrétními poli a tlačítky experimentu Cliqz. Tyto informace jsou spojena s unikátním identifikátorem Cliqz GmbH pro porozumění fungování služeb v průběhu času</li> </ul></p> <p>Mozilla sbírá telemetrické údaje včetně:<ul> <li>Počtu návštěv stránek jednotlivých vyhledávačů a které vyhledávače používáte</li><li>Unikátní identifikátor Cliqz, který Mozilla umožňuje dohledat spojitost mezi telemetrií z Firefoxu a systému Cluqz</li> </ul></p>
```

Comment:
Yeah, but that's known. Cliqz formerly broke the normalized format we had, and there isn't a safe way to preserve the work (since, as @clouserw has noted, just joining with commas won't work in some languages.